Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/SpatialGDKEditorInteropCodeGenerator.cpp

```diff
@@ -128,10 +132,22 @@ FString GenerateIntermediateDirectory()
 bool SpatialGDKGenerateInteropCode()
 {
 	const USpatialGDKEditorToolbarSettings* SpatialGDKToolbarSettings = GetDefault<USpatialGDKEditorToolbarSettings>();
-	ClassHeaderMap InteropGeneratedClasses;
-	for (auto& Element : SpatialGDKToolbarSettings->InteropCodegenClasses)
+	TArray<UClass*> InteropGeneratedClasses;
+	for (TObjectIterator<UClass> It; It; ++It)
 	{
-		InteropGeneratedClasses.Add(Element.ReplicatedClass, Element.IncludeList);
+		if (*It == UObject::StaticClass())
+		{
+			continue;
+		}
+
+		if (It->HasAnySpatialClassFlags(SPATIALCLASS_GenerateTypeBindings))
```

Comment:
Not a big deal, but you can remove the nested if via a `continue`. I wouldn't bother unless you're already changing this file.